### DISCUSSION TITLE
Please help me restore my account.

### DISCUSSION THREAD

**Discussion by rootgitman** [community/community#136275] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Respected community administrators:
Hi,I am a loyal fan of GitHub community. GitHub has given me a lot of help. Thank you very much.
Tonight, after uploading the CF project, My GitHub account was suddenly blocked and I was overwhelmed. So please ask the community administrator to help me restore my account number. If I violate the rules, please tell me, I will cooperate with you to correct. Thanks again!
My GitHub account: onerootman@gmail.com username:onerootman

<img width="602" alt="截屏2024-08-19 20 08 11" src="https://github.com/user-attachments/assets/ad372095-99f6-48bf-9121-a75aab5f13c8">

**Comment by BONY-SL** [comment]
Use the official GitHub Support page to submit a request. You can do this by going to 

https://support.github.com/request

**Reply by rootgitman** [reply]
Thanks bro, I've submitted a ticket but no one has responded!

**Reply by BONY-SL** [reply]
https://www.youtube.com/watch?v=PH9MaDO9sNk

Please follow this video

**Reply by rootgitman** [reply]
Thanks brother, I can't reset my password now, and the following question is prompted!
<img width="602" alt="截屏2024-08-19 20 08 11" src="https://github.com/user-attachments/assets/a6ccf45b-6db4-4b1b-afda-716a7e51fe7a">

**Reply by BONY-SL** [reply]
i will find solution and send message quikley

**Reply by BONY-SL** [reply]
https://www.youtube.com/watch?v=piEoh_gCURE

fllow this it not right support git hub

**Reply by rootgitman** [reply]
Thanks, brother. I submitted a ticket according to this operation, but there was no result, so today, I created a new account to report this issue! Are you a community administrator?
